docs/src/main/tut/iso.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+title:  "Iso"
+section: "optics"
+scaladoc: "http://julien-truffaut.github.io/Monocle/api/#monocle.PLens"
+source: "https://github.com/julien-truffaut/Monocle/blob/master/example/src/main/scala/monocle/example/LensExample.scala"
+pageSource: "https://raw.githubusercontent.com/julien-truffaut/Monocle/master/docs/src/main/tut/iso.md"
+---
+
+# Iso
+
+An `Iso` is an Optic which converts elements of type `A` into elements of type
+`B` without loss.
+
+Consider these two case classes:
+
+```tut:silent
+case class Person(name: String, firstName : String)
```

Comment:
This example might be more illustrative if the fields in `Person` are of different type e.g. `String` and `Int`